clio-server/src/test/scala/org/broadinstitute/clio/server/dataaccess/PersistenceDAOSpec.scala

```diff
@@ -42,27 +50,80 @@ class PersistenceDAOSpec
 
   it should "write metadata updates to storage using the upsertId" in {
     val dao = new MemoryPersistenceDAO()
-    val document =
-      DocumentMock.default.copy(
-        mockFilePath = Some(URI.create("gs://the-file"))
+
+    val key1Long = 1L
+    val key1String = "mock-key-1"
+    val key1 = ModelMockKey(key1Long, key1String)
+    val metadata1 = ModelMockMetadata(
+      Some(1.234),
+      Some(1234),
+      Some(OffsetDateTime.now()),
+      Some(Seq.empty[String]),
+      Some(Seq.empty[URI]),
+      Some(DocumentStatus.Normal),
+      Some('md5),
+      Some(URI.create("gs://the-file")),
+      Some(1234L)
+    )
+    val document1 = key1.asJson
+      .deepMerge(metadata1.asJson)
+      .deepMerge(
+        Map(
+          ElasticsearchIndex.UpsertIdElasticsearchName -> UpsertId
+            .nextId()
+        ).asJson
+      )
+      .deepMerge(
+        Map(
+          ElasticsearchIndex.EntityIdElasticsearchName -> Symbol(
```

Comment:
Doesn't need to be a `Symbol` any more, and should be able to get fused with above into a single `Map`.